packages/react-core/src/components/AboutModal/AboutModal.tsx

```diff
@@ -36,114 +38,59 @@ export interface AboutModalProps {
   disableFocusTrap?: boolean;
 }
 
-interface ModalState {
-  container: HTMLElement;
-}
-
-export class AboutModal extends React.Component<AboutModalProps, ModalState> {
-  static displayName = 'AboutModal';
-  private static currentId = 0;
-  private id = AboutModal.currentId++;
-  ariaLabelledBy = `pf-about-modal-title-${this.id}`;
-  ariaDescribedBy = `pf-about-modal-content-${this.id}`;
-
-  static defaultProps: PickOptional<AboutModalProps> = {
-    className: '',
-    isOpen: false,
-    onClose: (_e): any => undefined,
-    productName: '',
-    trademark: '',
-    backgroundImageSrc: '',
-    noAboutModalBoxContentContainer: false,
-    appendTo: null as HTMLElement
-  };
-
-  constructor(props: AboutModalProps) {
-    super(props);
-
-    this.state = {
-      container: undefined
-    };
-    if (props.brandImageSrc && !props.brandImageAlt) {
-      // eslint-disable-next-line no-console
-      console.error('AboutModal:', 'brandImageAlt is required when a brandImageSrc is specified');
-    }
+export const AboutModal: React.FunctionComponent<AboutModalProps> = ({
+  children,
+  className,
+  isOpen = false,
+  onClose = (_e): any => undefined,
+  productName = '',
+  trademark = '',
+  backgroundImageSrc = '',
+  brandImageSrc,
+  brandImageAlt,
+  noAboutModalBoxContentContainer = false,
```

Comment:
This could be a followup issue, but would we want to rename this prop to align with our other props? `hasAboutModalBoxContentContainer` (or shorten to `hasContentContainer`), and invert the logic (default to true, then inside AboutModalBoxContent apply the wrapper div when true and just the `children` when false)